.dockerignore

```diff
@@ -15,6 +15,7 @@ TAGS
 encoding/thrift/thriftrw-plugin-yarpc/thriftrw-plugin-yarpc
 encoding/x/protobuf/protoc-gen-yarpc-go/protoc-gen-yarpc-go
 internal/crossdock/crossdock
+internal/examples/env/
```

Comment:
Remove this in .gitignore - you probably have a clone from a long time ago, this is no longer generated.